src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-detail/tutorial-group-detail.component.ts

```diff
@@ -32,12 +30,22 @@ export class TutorialGroupDetailComponent implements OnChanges {
     course: Course;
     formattedAdditionalInformation?: SafeHtml;
 
-    faQuestionCircle = faQuestionCircle;
     readonly Math = Math;
 
     sessions: TutorialGroupSession[] = [];
 
-    tutorialDetailSections: DetailOverviewSection[];
+    tutorInitials: string;
+    tutorDefaultProfilePictureHue: string;
+    isMessagingEnabled: boolean;
+    utilization: number | undefined;
+
+    // Icons
+    faUserCheck = faUserCheck;
```

Comment:
Can we make the icons readonly here?

Reply:
Yes, done in this [commit](https://github.com/ls1intum/Artemis/pull/9445/commits/0de2b91ab2cda99670a2a48544bae7d0d3d46992). 👍🏼